.github/workflows/build_esp32.yml

```diff
@@ -66,7 +66,9 @@ jobs:
         - 'adafruit_qtpy_esp32s3'
         - 'bpi_leaf_s3'
         - 'espressif_esp32s3_devkitc_1'
+        - 'espressif_esp32s3_devkitc_1'
```

Comment:
This line is duplicated to above line of 68

Reply:
oops, removed now!